test/NuGet.Clients.Tests/NuGet.CommandLine.Test/RestoreNETCoreTest.cs

```diff
@@ -5,11 +5,15 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+using System.Reflection;
```

Comment:
any chance we can split this file? we can use partial classes to keep everything in the same class, so test history remains, if that's important. Otherwise we could make a `RestoreNETCore` folder/namespace and have more meaningful class names. However, I don't have any suggestions for a good way to split the tests. I just don't like a 8500 line file with so many tests.

Reply:
I don't necessarily have that issue with tests, but I'd want to discuss it team wide before making a decision like that. 

Also have stuff in one class makes it easy to copy templates :P 

Let's just discuss.